src/services/commit/useCommitCoverageDropdownSummary.tsx

```diff
@@ -43,6 +45,17 @@ const RepositorySchema = z.object({
   commit: z
     .object({
       compareWithParent: CompareWithParentSchema.nullable(),
+      uploads: z
+        .object({
+          edges: z.array(
+            z
```

Comment:
nit: can you oneline this

Reply:
This format is being overridden by the formatter, so I had to pull it out into it's own const.